exercises/perfect-numbers/description.md

```diff
@@ -7,13 +7,13 @@ The aliquot sum is defined as the sum of the factors of a number not including t
 For example, the aliquot sum of `15` is `1 + 3 + 5 = 9`.
 
 - **Perfect**: aliquot sum = number
-  - 6 is a perfect number because (1 + 2 + 3) = 6
-  - 28 is a perfect number because (1 + 2 + 4 + 7 + 14) = 28
+  - `6` is a perfect number because `1 + 2 + 3 = 6`
```

Comment:
Should these all have a trailing period?

Reply:
Probably not, the heading bullet in this nested group is not a sentence, though the colon could be moved.

I would maybe have done it like this:

```md
- **Perfect** aliquot is `sum = number`:
  - 6 is a perfect number because `1 + 2 + 3 = 6`
```

But I think it is pretty OK the way it is.

Reply:
"is" or "has" ?

~~~~text
**Perfect** aliquot is `sum = number`
**Perfect** aliquot has `sum = number`
~~~~

Reply:
It has the property of the sum and number being equal, so probably `has`.  It might also be clearer to use the words equals, rather than the symbol (as in math) which might be confused with the symbol (as in programming) which may not communicate the same way depending on the language.

Reply:
I'd suggest something like **Perfect**: when a `number` equals its `aliquot sum`

**Perfect** aliquot is `sum = number`
**Perfect** aliquot has `sum = number`
both read like we're calling a single aliquot perfect, but that wouldn't be a property ascribed to an aliquot or aliquot sum. The property belongs to `number` so we should make that clear.

Reply:
Thanks @BNAndras ! 🍬
Changes applied which required inverting the logic on "greater than" and "less than".